LibreNMS/Alert/Transport/Kayako.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Søren Friis Rosiak <sorenrosiak@gmail.com>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+namespace LibreNMS\Alert\Transport;
+
+use LibreNMS\Alert\Transport;
+
+class Kayako extends Transport
+{
+    public function deliverAlert($obj, $opts)
+    {
+        if (!empty($this->config)) {
+            $opts['url'] = $this->config['kayako-url'];
+            $opts['key'] = $this->config['kayako-key'];
+            $opts['secret'] = $this->config['kayako-secret'];
+            $opts['user'] = $this->config['kayako-user'];
+            $opts['department'] = $this->config['kayako-department'];
+        }
+        return $this->contactOsticket($obj, $opts);
+    }
+
+    public function contactOsticket($obj, $opts)
+    {
+        global $config;
+
+        $url   = $opts['url']."/Tickets/Ticket";
+        $key = $opts['key'];
+        $secret = $opts['secret'];
+        $user = $opts['user'];
+        $department = $opts['department'];
+        $ticket_type= 1;
+        $ticket_status = 1;
+        $ticket_prio = 1;
+        $salt = mt_rand();
+        $signature = base64_encode(hash_hmac('sha256',$salt,$secret,true));
```

Comment:
This needs spaces after the commas: https://travis-ci.com/librenms/librenms/jobs/140552481#L678